src/main/frontend/components/block.cljs

```diff
@@ -948,18 +948,18 @@
 
 (defn- render-macro
   [config name arguments macro-content format]
-  (if macro-content
-    (let [ast (->> (mldoc/->edn macro-content (gp-mldoc/default-config format))
-                   (map first))
-          paragraph? (and (= 1 (count ast))
-                          (= "Paragraph" (ffirst ast)))]
-      (if (and (not paragraph?)
-               (mldoc/block-with-title? (ffirst ast)))
-        [:div
-         (markup-elements-cp (assoc config :block/format format) ast)]
-        (inline-text format macro-content)))
-    [:span.warning {:title (str "Unsupported macro name: " name)}
-     (macro->text name arguments)]))
+  [:div {:class (str "macro " name)}
```

Comment:
A macro name can be any string, it could conflict with another class name in CSS.
I prefer using something like `data-macro-name="{name}" class="macro"`.
CC @xyhp915

Reply:
BTW, data attributes are selectable by CSS too.

Reply:
Also had this concern and glad it was addressed